build.sbt

```diff
@@ -12,7 +12,21 @@ lazy val syntactic = project
   .settings(
     moduleName := "syntactic",
     libraryDependencies ++= Seq(
-      scalameta
+      scalameta,
+      junit,
+      junitInterface
+    )
+  )
+
+lazy val syntacticWhitelist = project
```

Comment:
Is that a leftover?

Reply:
Yes... Removed in 3219f8f0903fa1e5646d0964e9917cd48a881675